clang/include/clang/Basic/Attr.td

```diff
@@ -3908,6 +3908,12 @@ def X86ForceAlignArgPointer : InheritableAttr, TargetSpecificAttr<TargetAnyX86>
   let Documentation = [X86ForceAlignArgPointerDocs];
 }
 
+def SectionMemtag : InheritableAttr {
+  let Spellings = [ClangGCC<"section_memtag">];
```

Comment:
maybe `force_memtag`? I don't think `section_memtag` is very meaningful if you don't know the context

Reply:
We could reuse `force_memtag` to e.g. force the stack instrumentation to not use stack safety analysis for a particular local variable and always tag it with MTE.